test/http_SUITE.erl

```diff
@@ -307,6 +307,20 @@ echo_body(Config) ->
 	end || Size <- lists:seq(MTU - 500, MTU)],
 	ok.
 
+%% Check if sending request whose header name is bigger than 64 bytes causes 431
+echo_body_max_header_name_length(Config) ->
+	ConnPid = gun_open(Config),
+	Ref = gun:post(ConnPid, "/echo/body", [{binary:copy(<<$a>>, 32768), <<"bad">>}], << "echo=name" >>),
```

Comment:
@essen I had to send a much larger header name because a binary with size right after the limit (which would be 65 bytes) was not triggering the error. Or I am missing something obvious or there is some buffer handling that is causing it to not check the whole header name.

The same applies to the header value below.